drivers/led/Kconfig.gpio

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config LED_GPIO
+	bool "GPIO LED driver"
+	depends on GPIO
```

Comment:
```suggestion
	depends on GPIO && $(dt_compat_enabled, gpio-leds)
```

Reply:
ok, added

Reply:
btw. `$(dt_compat_enabled,gpio-leds)` - space must be excluded